### DISCUSSION TITLE
Filter by tracker error

### DISCUSSION THREAD

**Discussion by anon1y4012** [transmission/transmission#4905] [Ideas]
I would love a feature that allows me to only show transfers that currently have a tracker error, denoted by the yellow caution sign. This would be helpful for torrents that have been trumped or deleted from the tracker and no longer need to seed. 

I see the "Error" option listed at the top, but I assume this is only for Transmission related errors such as "data file missing" and so on.

**Comment by kambala-decapitator** [comment]
yes, very useful feature

**Comment by dantearmok** [comment]
In what specific tracker errors are you interested?

From what I can tell, tracker errors are a dime a dozen, and they aren't all that specific...?

**Reply by anon1y4012** [reply]
Just an error in general. It'd be useful for deleting files that have been trumped, removed, etc. The specific error isn't relevant.

**Comment by kambala-decapitator** [comment]
any error. The motivation is ability to find torrents with errors quickly, e.g. add new sort field to make torrents with yellow triangle appear on top.

**Comment by dantearmok** [comment]
Any error?  Does that include when trackers don't talk back?  How do you tell the difference between trackers that are currently clogged vs trackers that have been taken offline?

I really think to even consider this, you need to be specific as to the errors.

**Reply by anon1y4012** [reply]
If you want to be specific that'd be great, but even being able to see all errors together would make the job of cleaning things up much faster. 

Say you have a list of 500 torrents, and 5 have errors, it's much easier to put all 5 together and then check why they have an error as opposed to scrolling through the list hunting and pecking.

**Reply by dantearmok** [reply]
I agree wrt the convenience factor.

But "all errors" makes no sense to me, and programmatically it can be a disaster - killing off torrents left and right.  If a torrent has two dozen trackers, and one or even a dozen returns "an error" - so what?  There are still dozens of trackers to cover for 'em!

So you need to narrow down the criteria.  Exactly what errors should result in the torrent being paused and pushed to the Error category?

**Reply by kambala-decapitator** [reply]
not sure why you want to kill/pause torrents. This feature is solely a visual feedback to the user to be able to quickly see/filter torrents with tracker errors.

**Reply by anon1y4012** [reply]
This wouldn't be an auto-kill feature, and for private trackers where there's only 1 tracker per torrent it makes perfect sense.

I suppose if you really wanted you could have keywords like "Trump", "Dupe", "Season Pack" etc.

**Comment by MatanZ** [comment]
[tremc](https://github.com/tremc/tremc) has this feature.

**Comment by Pentaphon** [comment]
https://github.com/openscopeproject/TrguiNG has this feature.
